src/components/dount/utils.tsx

```diff
@@ -0,0 +1,111 @@
+import * as d3 from "d3";
+import './styles.scss';
+import _ from 'lodash';
+import { ModifiedData } from "./types";
+import { format } from "d3";
+
+const createInnerCircle = (svg: d3.Selection<SVGGElement, unknown, null, undefined>, innerCircle: number) => {
```

Comment:
need to declare appropriate types

Reply:
Review it -> https://stackoverflow.com/questions/65695968/what-typescript-type-do-i-assign-d3-select-in-order-to-be-able-select-the-svg.